web/spacewalk-web.changes.mbussolotto.pam_default

```diff
@@ -0,0 +1 @@
+- Enable PAM Authentication by default
```

Comment:
Rather than adding a new changelog file could you merge and rewrite the other one as it hasn't been released yet? That would make more sense for users.

Reply:
this changelog would be required as well, since the original PR didn't have any change in `spacewalk-web` package https://github.com/uyuni-project/uyuni/pull/8537/files .